.dorpsgek.yml

```diff
@@ -4,11 +4,6 @@ notifications:
       - openttd
       - openttd.notice
 
-  push:
-    only:
-    - master
-    only-by:
-    - DorpsGek
```

Comment:
Expected change ?

Reply:
Yeah .. this should never have been here to start with. So I sneaked it in ;)

This exception is in OpenTTD repo .. and clearly people loved copy/pasting :D